collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/SaveAsTest.java

```diff
@@ -0,0 +1,32 @@
+package org.odk.collect.android.feature.formentry;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.support.CollectTestRule;
+import org.odk.collect.android.support.TestRuleChain;
+
+@RunWith(AndroidJUnit4.class)
+public class SaveAsTest {
```

Comment:
There was no coverage of the "Instance naming". Added this in to give myself some while I was moving things around.